pages/moderation-search/moderation-search.jsx

```diff
@@ -0,0 +1,250 @@
+import React from 'react';
```

Comment:
Same code as the old `search.jsx`. `search.jsx` was used by a few pages but not anymore. Due to changes in this PR, `moderation.jsx` becomes the only page that uses this component that's why I simply renamed it to `moderation-search.jsx`. I will file a follow-up ticket to clean up code in this file as we don't need so many if-else checks anymore since only one page uses this component.